packages/client/modules/teamDashboard/components/TeamDashHeader/TeamDashHeader.tsx

```diff
@@ -157,11 +158,12 @@ const TeamDashHeader = (props: Props) => {
         </Avatars>
       </TeamHeaderAndAvatars>
       <Tabs
-        activeIdx={isTasks ? 1 : 0}
+        activeIdx={isTasks ? 1 : 0 || isIntegrations ? 2 : 0}
```

Comment:
+1 I think it could be more readable, maybe something like:

```
const getActiveIndex = () => {
  if (isTasks) return 1;
  if (isIntegrations) return 2;
  return 0;
};
```